test/deploy/rackhd_ha_install.py

```diff
@@ -0,0 +1,217 @@
+"""
+Copyright © 2017 Dell Inc. or its subsidiaries.  All Rights Reserved. 
+
+This script prepares the VMs for an HA clustered configuration
+This is for setting up a cluster to do HA development and testing.
+
+
+This script performs the following functions:
+    - Configures the control network for each node in the cluster
+    - installs the DHCP configuration
+    - sets the /etc/hosts config
+    - sets the hostname
+
+usage:
+    python run_tests.py -stack <stack ID> -test deploy/rackhd_ha_install.py -numvms <num>
+"""
+from jinja2 import Environment, FileSystemLoader
+import os
+import sys
+import subprocess
+import unittest
+import time
+from nosedep import depends
+
+# set path to common libraries
+sys.path.append(subprocess.check_output("git rev-parse --show-toplevel", shell=True).rstrip("\n") + "/test/common")
```

Comment:
Okay if I fix that in a quick follow-on PR?  I currently don't need to rebase and it would be easier to do a separate PR.